ts/src/vendor-specific-values.ts

```diff
@@ -13,10 +20,14 @@ export const Chromium: Readonly<VendorSpecificValues> = {
     'https://publickeyservice.msmt.aws.privacysandboxservices.com',
     'https://publickeyservice.msmt.gcp.privacysandboxservices.com',
   ],
-  maxEventLevelChannelCapacityPerSource: {
+  maxEventLevelChannelCapacityPerSourceWithoutAttributionScopes: {
     [SourceType.event]: 6.5,
     [SourceType.navigation]: 11.5,
   },
+  maxEventLevelChannelCapacityPerSourceWithAttributionScopes: {
+    [SourceType.event]: 6.5,
+    [SourceType.navigation]: 11.55,
```

Comment:
Since this value isn't actually listed in [Chromium's parameters](https://github.com/WICG/attribution-reporting-api/blob/main/params/chromium-params.md) yet, maybe the PR description can explain the significance of 11.55 here vs the without-scopes 11.5.

Reply:
should we add this into the Chromium's parameters as well? or that requires a separate PR?

Reply:
That can wait until the Chromium implementation changes, but I would like for this PR to explain the origin of 11.55.

Reply:
ack.